Source/WebCore/html/HTMLElement.idl

```diff
@@ -34,6 +34,10 @@
     [CEReactions=Needed] attribute boolean translate;
     [CEReactions=Needed] attribute [AtomString] DOMString dir;
 
+    // TextTrackCue related
+    [Reflect] attribute boolean cue;
+    [Reflect] attribute boolean cuebackground;
```

Comment:
You need CEReactions=Needed for both.

Reply:
I was wondering about that. I guess it would make sense just to have them generically even if they are not really intended to be used that way.

Reply:
Yeah, otherwise modifications of these attributes wouldn't trigger callbacks which would violate custom elements. (I will grant that it's unlikely that anyone needs this, but that's not a good enough reason to introduce inconsistencies.)

Reply:
Agree. Added.